index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tribute Page</title>
+</head>
+<body>
+    <div id="main">
```

Comment:
It would be nice to use an HTML semantic tag like `main` instead of a general `div` tag. Using HTML semantic tags makes your code a lot more readable, identified by SEO.

Reply:
Nicely done